<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chart</class>
 <widget class="QDialog" name="Chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="calendarLayout">
     <item>
      <widget class="QDateEdit" name="calendar">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="latestButton">
       <property name="text">
        <string>Latest 24 Hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dayAgoButton">
       <property name="text">
        <string>-1 Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hoursAgoButton">
       <property name="text">
        <string>-3 Hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hoursLaterButton">
       <property name="text">
        <string>+ 3 Hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dayLaterButton">
       <property name="text">
        <string>+1 Day</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="chartArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>551</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="chartLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
